runmariadb.sh

```diff
@@ -12,6 +12,18 @@ if [ ! -d "${DATADIR}/mysql" ]; then
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_buffer_pool_size 5M
     crudini --set "$MARIADB_CONF_FILE" mysqld innodb_log_buffer_size 512K
 
+    # Config MariaDB to enable TLS
+    if [ -d "/certs" ] 
```

Comment:
you should maybe verify that they are all there individually, and error in case one would be missing. The current way might make it more difficult to debug if mariadb refuses to start due to a missing file.